File: semgrep-core/tests/go/metavar_equality_stmt.go
package main

import "fmt"

func main() {
    //ERROR:
    if (x > 2) {
        fmt.Println("hello world")
        fmt.Println("goodbye world")
    } else {
        fmt.Println("hello world")
        fmt.Println("goodbye world")
    }

    if (x > 2) {
        fmt.Println("hello world")
        fmt.Println("goodbye world")
    } else {
        fmt.Println("hello world")
    }
}

File: semgrep-core/tests/kotlin/metavar_equality_stmt.kt
class Foo {
    fun foo() {
        // ERROR:
        if (x > 2) {
            bar()
            foo()
        } else {
            bar()
            foo()
        }
   
        if (x > 2) {
            foo()
            bar()
        } else {
            foo()
        }
    }
}
